PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,119 @@
+# Quality Assurance Checklist
+To make reviews more efficient, please make sure the board meets the following standards and check everything off once the board meets the quality check. Once everything has been checked, the assigned reviewers will begin the review process. _Edit this description to check off the list._
+
+There are exceptions with all guidelines. As long as your decisions are justified, then you are good! Contact the reviewers or the leads about any exceptions.
+
+**Please *read* every word on *every* bullet point before checking off the corresponding box.**
+
+## Minimum Prerequisites
+- [ ] The board passes ERC and DRC.
+    - Note some ERC warnings are acceptable (**_CAUTION_** please be careful)
+    - Note some DRC warnings are acceptable (less so than ERC: **_CAUTION_** please be careful).
+- [ ] All traces are routed.
+- [ ] Units are in metric.
+- [ ] Gerbers are in a zip file in the main directory (after physical design is done)
+
+**Please *read* every word on *every* bullet point before checking off the corresponding box.**
+
+## Schematic Level Requirements 
+- [ ] Is proper noise resistance given to all peripheral devices (bypass caps and coils/ferrites)?  
+- [ ] Is proper ESD protection given to all MCU input pins (zener diodes)?  
```

Comment:
Perhaps the designer should leave a note on the input pin rating for the MCU (3.3V/5V) so the reviewer can check if the zener is of the appropriate voltage.